Wikipedia/Code/ArticlePlaceView.swift

```diff
@@ -3,14 +3,20 @@ import MapKit
 import WMF
 
 class ArticlePlaceView: MKAnnotationView {
-    private let imageView: UIImageView
-    private let selectedImageView: UIImageView
+    private let imageView: UIView
```

Comment:
Just curious... why doesn't ArticlePlaceView use IB? No worries and not a blocker, just wondering.